src/styles/sfds/index.scss

```diff
@@ -364,4 +355,19 @@
   .usa-tag {
     padding: units('05') units(1);
   }
+
+  /** Forms **/
+  .usa-combo-box__input,
+  .usa-input-group,
+  .usa-input,
+  .usa-textarea,
+  .usa-range,
+  .usa-select {
+    background-color: $theme-white;
+  }
+
+  .usa-combo-box__list,
+  .usa-combo-box__list-option {
+    color: $theme-black;
```

Comment:
Suggestion(non-blocking): Update these to use the vars, since you're adding new css. Though this might be covered by later stories, if so I'm fine leaving it for now.